app/assets/stylesheets/administrate/components/_pagination.scss

```diff
@@ -15,3 +15,15 @@
     font-weight: $bold-font-weight;
   }
 }
+
+.pagination-with-total {
+  margin-bottom: $base-spacing * 4;
```

Comment:
I don’t think we need this `margin-bottom` here, do we? `.pagination` already has the same `margin-bottom` applied on Line 2.